cumulus/polkadot-parachain/src/service.rs

```diff
@@ -529,61 +513,6 @@ impl<R> BuildOnAccess<R> {
 	}
 }
 
-/// Special [`ParachainConsensus`] implementation that waits for the upgrade from
-/// shell to a parachain runtime that implements Aura.
-struct WaitForAuraConsensus<Client, AuraId> {
```

Comment:
It was not used and the compiler did not complained about it being unused?

Reply:
I remember it's not been used for a long time, when I was fixing this stuff for the async backing, it was already the case. IIRC, the nightly toolchain complained about this, but not the stable one.

Reply:
Yes, the compiler didn't complain and all the required tests in the CI passed.